deploy/without-rbac.yaml

```diff
@@ -35,6 +35,12 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: STATSD_HOST
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            - name: STATSD_PORT
+              value: '8125'
```

Comment:
As we have discussed this PR as it is is not upstream friendly - as documented at https://github.com/Shopify/edgescale/issues/498 we wanna first get this running for us internally. I don't think this change is needed - we will have to delete it anyway when we make upstream PR because statsd should be optional at that time. We can document this env variables somewhere else then.

Reply:
Gotcha, but how do we expect that the agent data will be passed in in upstream? Would be not be using ENV vars to pass it in?

Reply:
we would but this document you edited is for getting starting with the _defaults_

Reply:
Yeah that makes sense 👍